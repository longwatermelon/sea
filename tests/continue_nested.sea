let write_buf:int = 0;

fn write_char(c:int) -> void {
    write_buf = c;
    syscall(4, 1, &write_buf, 1);
}

fn main() -> int {
    let i:int=0;
    while (i<3) {
        let j:int=0;
        while (j<5) {
            j=j+1;
            if (j==3) {
                continue;
            }
            write_char(i+48);
            write_char(j+48);
            write_char(32);
        }
        i=i+1;
    }
    return 0;
}
