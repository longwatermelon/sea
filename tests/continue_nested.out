RETURN_CODE:0
STDOUT:01 02 04 05 11 12 14 15 21 22 24 25 
